Python/marshal.c

```diff
@@ -305,6 +305,11 @@ w_bytecode(PyCodeObject *code, WFILE *p)
         }
         i += _PyOpcode_Caches[opcode];
     }
+    // Terminate with two zero bytes, so that programs scanning .pyc files can
+    // skip over the bytecode (even if they don't know the compression scheme).
+    // This is simpler than writing the compressed size in the header, which
+    // requires two loops (one to count the bytes, then one to write them):
+    w_short(0, p);
```

Comment:
Ugh, sorry, I can't stomach this. It feels certain that at some point in the future we'll have a corner case where we encounter \0\0 in the middle. If an extra pass is too expensive let's just go with what you had before. Or let's not do this -- I am feeling pretty lukewarm about this scheme, especially since the register VM will require a whole new approach anyway.